src/autogluon/bench/eval/scripts/run_evaluation_openml.py

```diff
@@ -150,15 +250,16 @@ def run(
         banned_datasets=banned_datasets,
         infer_batch_size=infer_batch_size,
         treat_folds_as_datasets=treat_folds_as_datasets,
+        clean_data=clean_data,
     )
 
     folds_to_keep = sorted(results_raw["fold"].unique())
 
-    if len(folds_to_keep) > 1:
+    if len(frameworks_run) > 1:
```

Comment:
It should be this:

`if len(folds_to_keep) > 1 and len(frameworks_run) > 1:`

Reply:
Thanks for pointing this out! `compute_win_rate_per_dataset()` had been updated to allow `folds==1`, is it intended?

Reply:
oh, if it works with folds==1 then you can ignore this.